js/script.js

```diff
@@ -737,58 +779,70 @@ async function mergeSettings() {
         // from hex strings to numbers
         structure: Object.keys(settings.structure).reduce((newObj, hexString) => {
             // new object, converted key (hex string -> numeric), same value
-            newObj[parseInt(hexString, 16)] = settings.structure[hexString]
+            newObj[parseInt(hexString, 16)] = settings.structure[hexString];
 
-            return newObj
+            return newObj;
         }, {})
-    }
+    };
 
     // merge with the defaults and send back
     return {
         ...BASE_SETTINGS,
         ...transformedSettings
-    }
+    };
 }
 
 async function programScript(stages) {
-    butProgram.disabled = true
-    butProgramNvm.disabled = true
+    butProgram.disabled = true;
+    butProgramNvm.disabled = true;
     try {
-        await fetchFirmwareForSelectedBoard()
-    } catch(error) {
-        errorMsg(error.message)
-        return
+        await fetchFirmwareForSelectedBoard();
+    } catch (error) {
+        errorMsg(error.message);
+        return;
     }
 
     // pretty print the settings object with VERSION placeholders filled
-    const settings = await mergeSettings()
-    const settingsString = JSON.stringify(settings, null, 2)
-    const strippedSettings = settingsString.replaceAll('VERSION', semver)
-    logMsg(`Flashing with settings: <pre>${strippedSettings}</pre>`)
+    const settings = await mergeSettings();
+    console.log(settings);
```

Comment:
Not sure if you're leaving console logs in intentionally? If not, there's usage further down.

Reply:
Yeah, I meant to remove.